### DISCUSSION TITLE
如何通过打包配置控制一些路由文件夹不被打包进去，该如何实现呢？

### DISCUSSION THREAD

**Discussion by NieShiyi** [web-infra-dev/rsbuild#1901] [General]
我采用的Menorepo和约定式路由组织项目
如何通过打包配置控制一些路由文件夹不被打包进去，该如何实现呢？或者有没有什么其他好的思路

**Comment by chenjiahan** [comment]
这两个话题是不是重复了：https://github.com/web-infra-dev/rsbuild/discussions/1900

**Reply by NieShiyi** [reply]
有点重复，那我关闭了吧
